app/views/orders/_update_status.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_for order, url: { controller: 'orders', action: 'update', id: order.id }, method: :patch do |form| %>
```

Comment:
can we just user order_path(order) in place of `{ controller: 'orders', action: 'update', id: order.id }`